lib/ansible/plugins/inventory/openstack.py

```diff
@@ -81,6 +81,11 @@
                 inventory script's option fail_on_errors)
             type: bool
             default: 'no'
+        all_projects:
```

Comment:
You need a `version_added: 2.10` for this.

Reply:
Good idea, I've added `version_added` in the latest commit, as well as the changelog fragment.